printrun/pronterface.py

```diff
@@ -904,6 +904,7 @@ def _add_settings(self, size):
         self.settings._add(BooleanSetting("clamp_jogging", False, _("Clamp manual moves"), _("Prevent manual moves from leaving the specified build dimensions"), "Printer"))
         self.settings._add(BooleanSetting("display_progress_on_printer", False, _("Display progress on printer"), _("Show progress on printers display (sent via M117, might not be supported by all printers)"), "Printer"))
         self.settings._add(SpinSetting("printer_progress_update_interval", 10., 0, 120, _("Printer progress update interval"), _("Interval in which pronterface sends the progress to the printer if enabled, in seconds"), "Printer"))
+        self.settings._add(BooleanSetting("cutting_as_extrusion", False, _("Display cutting moves"), _("Show moves where spindle is active as printing moves"), "Printer"))
         self.settings._add(ComboSetting("uimode", _("Standard"), [_("Standard"), _("Compact"), ], _("Interface mode"), _("Standard interface is a one-page, three columns layout with controls/visualization/log\nCompact mode is a one-page, two columns layout with controls + log/visualization"), "UI"), self.reload_ui)
```

Comment:
I would also support this being true by default - @IgorYeremin what do you think?

Reply:
Sure, I'm ok with making it on by default. I made it opt-in in case someone was abusing spindle commands for controlling something else on their printer. That should be a very small group of people and they can uncheck the box.

Reply:
@IgorYeremin Great, can you make the change and then I'll merge?